includes/widgets/icon-box.php

```diff
@@ -332,6 +343,40 @@ protected function register_controls() {
 			]
 		);
 
+		$this->add_group_control(
+			Group_Control_Box_Shadow::get_type(),
+			[
+				'name' => 'icon_box_hover_shadow',
```

Comment:
Here you will see this suffix `..._hover_shadow` where the established convention in all the widgets is `..._shadow_hover` (`_hover` always on the end).

Can you check the PR for the other occurances too please. A search and replace should find them all, but def worth confirming each change works and doesn't create duplicate names. It shouldnt thoygh.